### PR TITLE
Alpha Bug Fix (Result Display, DG, Check)

### LINKED ISSUE #138: Result Display Bugs
### 2) Clients plural for 1 Client.
Screenshot 2024-10-26 at 6 07 06 PM

### 3) Change Parameters and Examples to start on a new line.
Screenshot 2024-10-26 at 6 11 45 PM

### LINKED ISSUE #139: DG Bug
### Typo "ViewClientCommandParse" should be "ViewClientCommandparser".
Screenshot 2024-10-26 at 6 21 49 PM

### Should split Note into two seperate Notes.
Screenshot 2024-10-26 at 6 23 31 PM

### LINKED ISSUE #144: Help feature link Bug
### Wrong URL provided.
Screenshot 2024-10-26 at 8 16 37 PM

### LINKED ISSUE #146: Check not preserved after Edit Bug
Bug where a client checked in would no longer be checked in after editing.

### PR DESCRIPTION
**Pull request by xhamyo**
# Fixes #138

### New Line for Parameters and Examples.
Screenshot 2024-10-26 at 8 58 23 PM

### Client(s) instead of Client.
Screenshot 2024-10-26 at 8 59 42 PM

# Fixes #139 

# Fixes #144 

### Updated link and added command references.
Screenshot 2024-10-26 at 9 09 48 PM

# Fixes #146

### CODE REVIEW

File: src/main/java/seedu/address/model/car/CarMake.java [hunk 1]
```diff
@@ -9,10 +9,8 @@
  */
 public class CarMake {
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Car make should only contain alphanumeric characters, "
-                    + "with the first character being a capital letter "
-                    + "and it should not contain spaces nor be blank.";
+    public static final String MESSAGE_CONSTRAINTS = "Car make should only contain alphanumeric characters, "
+            + "with the first character being a capital letter, and it should not contain spaces nor be blank.";
```
Comment (SeanWong2) [inline src/main/java/seedu/address/model/car/CarMake.java:13]: Is this indentation level correct?

**Review by SeanWong2 — COMMENTED** [review]
LGTM

**Review by calebyyy — APPROVED** [review]
Very good use of encapsulation and meticulous details. Amazing work as always

**Review by limyuquan — APPROVED** [review]
lgtm

**Pull request merged**